examples/pipelines/stabilize_images.pipe.in

```diff
@@ -22,252 +22,304 @@ process input
 process stabilize
   :: stabilize_image
 
-# Algorithm to use for 'feature_tracker'.
+# Algorithm to use for 'track_features'.
 # Must be one of the following options:
-#   - default
-  feature_tracker:type = default
+#   - core
+#   - ocv_KLT
+    track_features:type = core
 
+  block track_features:core
 # Algorithm to use for 'descriptor_extractor'.
 # Must be one of the following options:
-#   - ocv
+#   - ocv_BRIEF
+#   - ocv_DAISY
+#   - ocv_FREAK
+#   - ocv_LATCH
+#   - ocv_LUCID
+#   - ocv_BRISK
+#   - ocv_ORB
+#   - ocv_SIFT
+#   - ocv_SURF
 #   - viscl
-  feature_tracker:default:descriptor_extractor:type = ocv
+    descriptor_extractor:type = ocv_SURF
 
 # The OpenCV cv::Algorithm type to use for 'extractor'.
-  feature_tracker:default:descriptor_extractor:ocv:extractor:type                              = Feature2D.SURF
-  feature_tracker:default:descriptor_extractor:ocv:extractor:Feature2D.SURF:extended           = false
-  feature_tracker:default:descriptor_extractor:ocv:extractor:Feature2D.SURF:hessianThreshold   = 5000
-  feature_tracker:default:descriptor_extractor:ocv:extractor:Feature2D.SURF:nOctaveLayers      = 3
-  feature_tracker:default:descriptor_extractor:ocv:extractor:Feature2D.SURF:nOctaves           = 4
-  feature_tracker:default:descriptor_extractor:ocv:extractor:Feature2D.SURF:upright            = true
+    block descriptor_extractor:ocv_SURF
+      extended           = false
+      hessian_threshold  = 5000
+      n_octave_layers    = 3
+      n_octaves          = 4
+      upright            = true
```

Comment:
We might consider setting this false by default.  This makes better matches if the images never rotate, but it will kill matches if the image rotates.